dataactcore/migrations/versions/a97dabbd44f4_drop_filetype_validation_table.py

```diff
@@ -21,14 +21,6 @@
 
 Session = sessionmaker()
 
-class FileTypeValidation(Base):
-    __tablename__ = 'file_type_validation'
-
-    file_id = sa.Column(sa.Integer, primary_key=True)
-    name = sa.Column(sa.Text)
-    description = sa.Column(sa.Text)
-    file_order = sa.Column(sa.Integer, nullable=False, server_default="0")
-
```

Comment:
What happens if we _do_ declare this class outside of the downgrade? If there's an explosion, we may need to update some other migrations 😅